build/vsts-ci.yml

```diff
@@ -38,6 +44,66 @@ phases:
       artifactName: PackageAssets
       artifactType: container
 
+################################################################################
+- phase: Linux arm
+################################################################################
+  variables:
+    BuildConfig: Release-netcoreapp3_1
+    OfficialBuildId: $(BUILD.BUILDNUMBER)
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+    DOTNET_MULTILEVEL_LOOKUP: 0
+  queue:
+    name: DotNet-Build
+    demands:
+    - agent.os -equals linux
+    container: UbuntuCrossArmContainer
+  steps:
+  - script: ./restore.sh
+    displayName: restore all projects
+  - script: ./build.sh -configuration $(BuildConfig) /p:CopyPackageAssets=true /p:SkipRIDAgnosticAssets=true -projects $(Build.SourcesDirectory)/src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj
+    displayName: build redist
+  # Only build native assets to avoid conflicts.
+  - script: ./build.sh -configuration $(BuildConfig) -projects $(Build.SourcesDirectory)/src/Native/Native.proj /p:TargetArchitecture=arm /p:CopyPackageAssets=true
+    displayName: Build
```

Comment:
>Build [](http://example.com/codeflow?start=17&length=5)

Should display name be: Build native assets

Reply:
I kept it at that because that is what we call it on the other native builds, but I think that makes more sense. Let me switch them all.